main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	"URezL/api"
+)
+
+func main() {
+	router := mux.NewRouter()
+	router.HandleFunc("/linkCut", api.PostLinkCut).Methods(http.MethodPost)
```

Comment:
REST APIs usually make use of a single entity route like "/links" to implement all CRUD operations with it. Please consider simplifying your set of routes.